bin/windows-toolset.ps1

```diff
@@ -40,22 +40,28 @@ if (!(Get-Command choco -ErrorAction SilentlyContinue)) {
     & $scriptDirectory\install-chocolatey.ps1
 }
 
-Install-Choco 7zip.commandline 9.20.0.20150210
+Install-Choco 7zip.commandline 9.20.0.20150210 'https://www.myget.org/F/puppetlabs'
```

Comment:
@Iristyle comments have been addressed in https://github.com/puppetlabs/puppet-agent/pull/457, but I have yet to send it through an automated run